### PR TITLE
Fix missing fastutf8

### PR DESCRIPTION
**Pull request by sbzlzh**
image
Fixed missing fastutf8, found that there seems to be something missing to introduce it correctly causing the error in the picture to occur when pressing F1 or pressing TAB after already using Chinese.

### CODE REVIEW

**Comment by TimGoll** [comment]
I guess @saibotk didn't acutally test his PR 👀 :D

**Review by TimGoll — APPROVED** [review]

File: gamemodes/terrortown/gamemode/shared/sh_init.lua [hunk 1]
```diff
@@ -577,6 +577,7 @@ include("ttt2/libraries/playermodels.lua")
 include("ttt2/libraries/entspawnscript.lua")
 include("ttt2/libraries/bodysearch.lua")
 include("ttt2/libraries/keyhelp.lua")
+include("ttt2/libraries/fastutf8.lua")
```
Comment (saibotk) [inline gamemodes/terrortown/gamemode/shared/sh_init.lua:580]: This should be the first in this block

**Review by saibotk — CHANGES_REQUESTED** [review]

**Comment by saibotk** [comment]
I did but seems like i missed selecting this change for the commit

File: gamemodes/terrortown/gamemode/shared/sh_init.lua (hunk 1 above)
Comment (sbzlzh) [reply gamemodes/terrortown/gamemode/shared/sh_init.lua:580]: all right

**Comment by TimGoll** [comment]
@saibotk done, you can approve it

**Review by saibotk — APPROVED** [review]

**Pull request merged**